src/coreclr/jit/importer.cpp

```diff
@@ -3938,6 +3889,11 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
         mustExpand = true;
     }
 
+    if ((ni == NI_System_ByReference_ctor) || (ni == NI_System_ByReference_get_Value))
```

Comment:
Could you please also add the raw handle intrinsics here? They have to be marked must-expand as well.